From chihau@gmail.com  Fri Feb 26 16:35:23 2010
From: Chihau Chau <chihau@gmail.com>
Date: Thu, 25 Feb 2010 20:20:09 -0300
Subject: Staging: dream: camera: sk5k3e2fx: fix code style issues
To: gregkh@suse.de
Cc: pavel@ucw.cz, linux-kernel@vger.kernel.org, devel@driverdev.osuosl.org, Chihau Chau <chihau@gmail.com>
Message-ID: <1267140009-18775-1-git-send-email-chihau@gmail.com>


From: Chihau Chau <chihau@gmail.com>

This fixes some code style issues about assignments in if conditions.

Signed-off-by: Chihau Chau <chihau@gmail.com>
Acked-by: Pavel Machek <pavel@ucw.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/dream/camera/s5k3e2fx.c
+++ b/drivers/staging/dream/camera/s5k3e2fx.c
@@ -743,12 +743,12 @@ static int s5k3e2fx_sensor_open_init(const struct msm_camera_sensor_info *data)
 	}
 
 	/* initialize AF */
-	if ((rc = s5k3e2fx_i2c_write_b(s5k3e2fx_client->addr,
-			0x3146, 0x3A)) < 0)
+	rc = s5k3e2fx_i2c_write_b(s5k3e2fx_client->addr, 0x3146, 0x3A);
+	if (rc < 0)
 		goto init_fail1;
 
-	if ((rc = s5k3e2fx_i2c_write_b(s5k3e2fx_client->addr,
-			0x3130, 0x03)) < 0)
+	rc = s5k3e2fx_i2c_write_b(s5k3e2fx_client->addr, 0x3130, 0x03);
+	if (rc < 0)
 		goto init_fail1;
 
 	goto init_done;
-- 
1.5.6.3

